pkg/operator/controllers/guardrails/policies/gktemplates-src/aro-deny-clusterversion-upgradeconfig/sync.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
+kind: Config
+metadata:
+  name: config
+  namespace: "openshift-azure-guardrails"
+spec:
+  match:
+    - excludedNamespaces:
+        [
+          "kube-*",
+          "openshift-kube-*",
+          "openshift",
+          "openshift-etcd*",
+          "openshift-monitoring",
+          "default",
+          "gatekeeper-system",
+          "openshift-apiserver*",
+          "openshift-authentication*",
+          "openshift-logging",
+          "openshift-redhat-marketplace",
+          "openshift-operators",
+          "openshift-user-workload-monitoring",
+          "openshift-pipelines",
+          "openshift-marketplace",
+          "openshift-multus",
+          "openshift-network*",
+          "openshift-vsphere-*",
+          "openshift-config-*",
+          "openshift-console",
+          "openshift-service-ca*",
```

Comment:
IIUC, as per [gk doc](https://open-policy-agent.github.io/gatekeeper/website/docs/exempt-namespaces/) this is excluding priv'd namespaces from gatekeeper's control, which will left them unprotected by our priv'd ns policy, isn't it?

Reply:
can you please test this out, by checking after this applied if the priv'd ns policy still work?